src/util/config.rs

```diff
@@ -1089,6 +1097,7 @@ mod test {
             (0, 0, "0s"),
             (0, 1, "1ms"),
             (2, 0, "2s"),
+            (24 * 3600, 0, "24h"),
```

Comment:
Should add day test case.

Reply:
@Connor1996 How about add a test case like (3600 * 24, 0, "1d")?